content/docs/reference-events.md

```diff
@@ -244,17 +244,17 @@ string pointerType
 boolean isPrimary
 ```
 
-A note on cross-browser support:
+Catatan untuk dukungan lintas _browser_:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+_Event_ penunjuk belum didukung oleh semua browser (saat penulisan artikel ini, _browser_ yang mendukung terdiri atas: Chrome, Firefox, Edge, dan Internet Explorer). React tidak mengisi dukungan untuk _browser_ lain karena akan meningkatkan ukuran berkas `react-dom` secara singnifikan.
 
-If your application requires pointer events, we recommend adding a third party pointer event polyfill.
+Jika aplikasi Anda memerlukan _event_ pointer, kami merekomendasikan Anda menambah _event_ penunjuk dari pihak ketiga.
```

Comment:
```suggestion
Jika aplikasi Anda memerlukan _pointer event_ , kami merekomendasikan Anda untuk menambah _polyfill_ untuk _pointer event_ dari pihak ketiga.
```

What do you think? 😄

Reply:
:+1, since "pointer event" isn't translated, I do think that "focus event" should also be retained as it is. wdyt?